aaelf64/aaelf64.rst

```diff
@@ -1240,14 +1240,17 @@ The following tables record single instruction relocations and relocations that
-  +============+============+====================+============+=========================================================================================================================+
-  | 307        | \-         | R\_<CLS>\_GOTREL64 | S+A-GOT    | Write bits [63:0] of X at byte-aligned place P.  This represents a 64-bit offset relative to the GOT.                   |
-  +------------+------------+--------------------+------------+-------------------------------------------------------------------------------------------------------------------------+
-  | 308        | \-         | R\_<CLS>\_GOTREL32 | S+A-GOT    | Write bits [31:0] of X at byte-aligned place P.  This represents a 32-bit offset relative to GOT, treated as signed;    |
-  |            |            |                    |            | Check that -2\ :sup:`31` <= X < 2\ :sup:`31`.                                                                           |
-  +------------+------------+--------------------+------------+-------------------------------------------------------------------------------------------------------------------------+
+  +------------+------------+----------------------+------------------+-------------------------------------------------------------------------------------------------------------------------+
+  | ELF64 Code | ELF32 Code | Name                 | Operation        | Comment                                                                                                                 |
+  +============+============+======================+==================+=========================================================================================================================+
+  | 307        | \-         | R\_<CLS>\_GOTREL64   | S+A-GOT          | Write bits [63:0] of X at byte-aligned place P.  This represents a 64-bit offset relative to the GOT.                   |
+  +------------+------------+----------------------+------------------+-------------------------------------------------------------------------------------------------------------------------+
+  | 308        | \-         | R\_<CLS>\_GOTREL32   | S+A-GOT          | Write bits [31:0] of X at byte-aligned place P.  This represents a 32-bit offset relative to GOT, treated as signed;    |
+  |            |            |                      |                  | Check that -2\ :sup:`31` <= X < 2\ :sup:`31`.                                                                           |
+  +------------+------------+----------------------+------------------+-------------------------------------------------------------------------------------------------------------------------+
+  | 315        | \-         | R\_<CLS>\_GOTPCREL32 | G(GDAT(S+A))- P  | Write bits [31:0] of X at byte-aligned place P.  This represents a 32-bit offset relative to GOT entry for an address, treated as signed;    |
```

Comment:
Although difficult to see unless you have a wide editor the Comment column is too long and the RST treats this as a malformed table.

I think 
```
  | 315        | \-         | R\_<CLS>\_GOTPCREL32 | G(GDAT(S+A))- P  | Write bits [31:0] of X at byte-aligned place P.  This represents a 32-bit offset relative to GOT entry for an address,  |
  |            |            |                      |                  | treated as signed; Check that -2\ :sup:`31` <= X < 2\ :sup:`31`.                                                        |
```
Will fix it. 

There's a script in tools/common/check-rst-syntax.sh which will run rstcheck over all the documents. Will be worth running that first to check.